src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,6 +1,45 @@
 import React from 'react';
 import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import { PreparedPost } from '../../app.typedef';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+interface Props {
+  post: PreparedPost;
+}
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
```

Comment:
same here you can prevent creating this variable by moving the object to the argument of the component